src/h.js

```diff
@@ -1,20 +1,32 @@
-export default function(tag, data) {
+function addChild(children, val) {
+  if (Array.isArray(val)) {
+    for (var j = 0; j < val.length; j++) {
+      children.push(val[j])
+    }
+  }
+  else if (val && val !== true) {
```

Comment:
I left you a comment on the other diff about this.

Basically, you are forgetting about the number 0.

```js
} else if (val != null && val !== true && val !== false) {
```